### ISSUE TITLE
Can I both training and testing?

### ISSUE THREAD

**Issue by lmolhw5252** [emedvedev/attention-ocr#90]
Hi I got a problem when I using your code, that my model is over fitting.
I want to have a test when I training my model.
So how can I use the code?

**Comment by emedvedev** [comment]
If your model is overfitting, try using a larger dataset, augmenting it, or just stopping earlier. That would be the most basic advice, I guess. If that doesn't help, you'd have to try adjusting hyperparameters or even changing the model a little, but unfortunately, I can only help with the project issues and bugs, not with general ML questions. If you are sure there's a bug in the codebase that inherently causes overfitting, that'd be a whole other issue, of course.

If you feel like you're completely stuck, try asking on Stack Overflow, there's quite a few people willing to help with ML there.

**Comment by lmolhw5252** [comment]
Thank you for your reply. 
I'm not problem about overfit, I just want to add predict valid set to watch real time accuracy.

**Comment by emedvedev** [comment]
I'm afraid `aocr` doesn't have the functionality you're looking for, so you could either write a wrapper script on top or fork it and modify the code. If you end up with something that could be of use to other people, please submit a PR!
